.github/workflows/ci.yml

```diff
@@ -54,6 +54,28 @@ jobs:
           name: python-package
           path: dist
 
+  codecov_coverage:
+    name: Codecov Coverage Report
+    needs: 
+    - build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code 🛎
+        uses: actions/checkout@v2
+      - name: Setup Python 🐍
+        uses: actions/setup-python@master
+        with:
+          python-version: 3.7
+      - name: Generate Report 📄
+        run: |
+          pip install pytest-cov
+          pytest --cov=./
```

Comment:
I think we should avoid running our test suite twice, see the job above. So maybe include the coverage generation/uploading in the job above.